scripts/install_plugins.py

```diff
@@ -63,11 +63,11 @@
   owd = os.getcwd()
   cwd = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
   os.chdir(cwd)
-  subsOut = os.popen('git config --file .gitmodules --name-only --get-regexp path').read()
+  subsOut = [x.split(' ')[0] for x in os.popen('git config --file .gitmodules --get-regexp path').read().split("\n")]
```

Comment:
Do we need to specify the empty space for the "x.split(' ')" here? Can we just use the default, i.e., "x.split()"?

Reply:
Yes, the default would work, tho' that would split on all whitespace instead of just spaces.